CONTRIBUTING.md

```diff
@@ -49,34 +41,27 @@ If approved by at least two core team members and there are no outstanding reser
 Once accepted the author of the RFC and their team can scope the work within their regular process. Link or reference the RFC in the related JIRA ticket.
 The core team will respond with design feedback, and the author should be prepared to revise it in response.
 
-**FAQ**
-
+#### FAQ
 Q: What qualifies as a substantive change?
-
 A: There is no strict definition, however examples of substantive changes are:
```

Comment:
It seems that these Q & A are now on the same line, i.e.:

> Q: Question A: Answer

Instead of

> Q: Question
> A: Answer

It seems the single dash (`-`) removed from each line here was breaking these onto separate lines. I think an empty line between the question and answer would also suffice in rendering the question and answer on separate lines, too.